newIDE/app/src/ResourcesList/LocalResourceExternalEditors.js

```diff
@@ -17,6 +18,14 @@ const editors: Array<ResourceExternalEditor> = [
     editDisplayName: t`Edit with Piskel`,
     kind: 'image',
     edit: options => {
+      const storageProvider = options.getStorageProvider();
+      if (storageProvider.internalName !== 'LocalFile') {
+        Window.showMessageBox(
```

Comment:
May I suggest you use the showAlert() for all those warnings.

Reply:
Would unfortunately require to pass the function as parameter, and one usage is in SpritesList which is a class component, so can't use the context. Not taking the risk to refactor this now!

Reply:
(and ResourcesSelector too actually!)